package.json

```diff
@@ -219,11 +219,13 @@
     }
   },
   "apiVersionMetadata": {
-    "comment": "Refer to ApiVersionMetadata in orgUtils.ts for details",
+    "comment1": "Refer to ApiVersionMetadata in orgUtils.ts for more details.",
```

Comment:
Nit: this is just organizational semantics to a facility that has no formal support. That said, it might be nice to have a more descriptive structure for these comments.
```
    "x-apiVersionMetadata-comments": [
        "Refer to ApiVersionMetadata in orgUtils.ts for more details.",
        "target section defines the dev server version (matchingDevServerVersion) and the API version that it can support (versionNumber)."
    ],
```

Reply:
This can really be anything that JSON can support (and that doesn't interfere with the data structure otherwise). `x-*` fields are a typical way to express project-specific information in "headers", so I threw that idea out there.

Reply:
Oh I like this... will change.